### ISSUE TITLE
Deploy multiple services with same task definition

### ISSUE THREAD

**Issue by msalinasboz** [aws-actions/amazon-ecs-deploy-task-definition#420]
Hi! we use task definitions to deploy many services, but at the moment when the task definition is deployed, each service has a different version of it, we expect that the previous task definitions created by de action be the same for all the services, otherways in ecs we have a lot of task definition with the same definition but different number of task.

![taskdeifnition](https://user-images.githubusercontent.com/99291435/204071322-2132a286-775b-4f2c-bbd3-27618cb90669.PNG)

Some idea what is happen or what we are doing in the wrong way?

**Comment by varunchillara27** [comment]
Are you using strategy matrix to pull in multiple images from ECR to ECS? I am doing something similar and am in the same situation, where multiple tasks

**Comment by msalinasboz** [comment]
No, im building the image in the job Build Image, then i take the out image and use it on task definition
